__tests__/integration.test.js

```diff
@@ -66,6 +66,32 @@ describe('GET: api/articles/:article_id', () => {
     });
 });
 
+describe('GET: api/articles', () => {
+    test('200: responds successfully with a list of all articles with a comment_count property', () => {
```

Comment:
You may just want to edit the title to also state "...and responds with the data sorted by data in descending order". 👍